src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
+
+  checkHealth() {
+    if (this.health <= 0) {
+      Animal.alive = Animal.alive.filter((animal) => animal !== this);
```

Comment:
The `checkHealth` method correctly checks if the animal's health is less than or equal to zero. However, ensure that the animal is removed from the `Animal.alive` array only when this condition is met. The current implementation already does this, so no changes are needed here.